data/blog/electron-3-0.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Electron 3.0.0
+author: codebytere
+date: '2018-09-20'
+---
+
+The Electron team is excited to annouce that the first stable release of Electron `v3.0` is now
+available from [electronjs.org](https://electronjs.org/) and via `npm install electron@latest`! It's jam-packed with upgrades, fixes, and new features, and we can't wait to see what you build with them. Below are details about this release, and we welcome your feedback as you explore.
+
+---
+
+## Release Process
+
+As we undertook development of `v3.0.0`, we sought to more empirically define criteria for a stable release by formalizing the feedback progress for progressive beta releases. `v3.0.0` would not have been possible without our [App Feedback Program](LINK-TBD) partners, who provided early testing and feedback during the beta cycle. Thanks to Atlassian, Atom, Microsoft Teams, Oculus, OpenFin, Slack, Symphony, VS Code, and other program members for their work. If you'd like to participate in future betas, please mail us at [info@electronjs.org](mailto:info@electronjs.org).
+
+## Changes / New Features
+
+Major bumps to several important parts of Electron's toolchain, including Chrome `v66.0.3359.181`, Node `v10.2.0`, and V8 `v6.6.346.23.`
+
+* [[#12656](https://github.com/electron/electron/pull/12656)] feat: `app.isPackaged`
+* [[#12652](https://github.com/electron/electron/pull/12652)] feat: `app.whenReady()`
+* [[#13183](https://github.com/electron/electron/pull/13183)] feat: `process.getHeapStatistics()`
+* [[#12485](https://github.com/electron/electron/pull/12485)] feat: `win.moveTop()` to move window z-order to top
+* [[#13110](https://github.com/electron/electron/pull/13110)] feat: TextField and Button APIs
+* [[#13068](https://github.com/electron/electron/pull/13068)] feat: netLog API for dynamic logging control
+* [[#13539](https://github.com/electron/electron/pull/13539)] feat: enable `webview` in sandbox renderer
+* [[#14118](https://github.com/electron/electron/pull/14118)] feat: `fs.readSync` now works with massive files
+* [[#14031](https://github.com/electron/electron/pull/14031)] feat: node `fs` wrappers to make `fs.realpathSync.native` and `fs.realpath.native` available
+
+## Breaking API changes
+
+* [[#12362](https://github.com/electron/electron/pull/12362)] feat: updates to menu item order control
+* [[#12991](https://github.com/electron/electron/pull/12991)] refactor: aliased arm builds are no longer published
```

Comment:
I get that this is technically a breaking change, but still... this should be moved to the bottom of the list :smile:

Reply:
i think i'll remove all chore-type PRs, they're not user facing so i doubt anyone will miss them 😆